config/environments/production.rb

```diff
@@ -22,36 +22,29 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
```

Comment:
I'm not certain but this might have been a customization in the past. We might need to restore this later if the site's JS and CSS isn't being precompiled. Test by attempting to compile and run in production environment. Just a reminder note if we see any issues. Hopefully not.

Reply:
Uglifier is not used with Rails 6, it was removed from the Gemfile so we shouldn't have it in settings either